web_ui/static/css/theme.css

```
@@ -226,36 +969,1919 @@ a:hover {
+    in srgb,
+    var(--ua-light-panel-start) 62%,
+    #f5f5f4
+  );
+}
+
+.ua-upload-page {
+  background:
+    radial-gradient(circle at 100% 0%, var(--ua-atmosphere), transparent 30rem),
+    var(--ua-upload-bg) !important;
+  color: var(--ua-upload-text);
+  height: var(--ua-upload-viewport-height, 100vh) !important;
```

Stylelint (17.14.0)

[error] Duplicate property "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)